docs/man/rpmbuild.1.scd

```diff
@@ -202,11 +148,6 @@ The following options may also be used:
 	but *%builddir*/*%buildsubdir* points back to the current working
 	directory. *%prep* is skipped entirely.
 
-*--target* _PLATFORM_
```

Comment:
This seems to have been dropped completely (not moved added elsewhere), is that intentional?

Reply:
Thanks for spotting, certainly not intentional.